content/windows.md

```diff
@@ -563,11 +396,44 @@ Depending on how you installed Windows and which firmware your computer is runni
 - Press the `+` key (or `SHIFT` + `=`) to increase the number of seconds the system will wait before booting the default OS; press `-` to decrease the number of seconds.
 - Once a timer is set, the menu will display on every boot without having to hold down `SPACE`. (To reverse this, press `-` until the timer is back to zero.)
 
-## Repairing the Pop!_OS Bootloader
+##### Add Windows to systemd-boot
+
+If you installed Windows on a [dedicated drive](#installing-on-a-dedicated-drive), then the systemd-boot menu will not recognize the Windows drive right away. To make systemd-boot aware of the Windows installation, we will need to copy the Windows boot files from the Windows boot partition, to the Pop!\_OS boot partition.
+
+1. Boot to Pop!_OS
```

Comment:
Escape character missing in Pop!_OS, also it appears periods are used inconsistently for the following list.